pkg/controller/egress/validate.go

```diff
@@ -53,6 +54,31 @@ func (c *EgressController) ValidateEgress(review *admv1.AdmissionReview) *admv1.
 		if len(newEgress.Spec.ExternalIPPools) > 0 {
 			return false, "spec.externalIPPools is not supported yet"
 		}
+		// Validate Egress trafficShaping
+		if newEgress.Spec.Bandwidth != nil {
+			pattern := "^[1-9][0-9]*[KMG]$"
+			if matched, _ := regexp.MatchString(pattern, newEgress.Spec.Bandwidth.Rate); !matched {
+				return false, fmt.Sprintf("Rate %s in Egress %s is Invalid. Expected format is [integer + K(Kbps), M(Mbps) or G(Gbps)]. e.g. 300K, 100M, 2G", newEgress.Spec.Bandwidth.Rate, newEgress.Name)
+			}
+			if matched, _ := regexp.MatchString(pattern, newEgress.Spec.Bandwidth.Burst); !matched {
+				return false, fmt.Sprintf("Burst %s in Egress %s is Invalid. Expected format is [integer + K(Kbps), M(Mbps) or G(Gbps)]. e.g. 300K, 100M, 2G", newEgress.Spec.Bandwidth.Burst, newEgress.Name)
+			}
```

Comment:
Reuse `resource.ParseQuantity` to validate it

Reply:
Done.